### ISSUE TITLE
Line numbers in stylish output are non-sensical.

### ISSUE THREAD

**Issue by adrianhall** [sasstools/sass-lint#466]
I am linting the following file:

```
html,
body {
    padding: 0;
    margin: 0;
    font-size: 16px;
}

// View Imports
@import './views/Application';
```

This is 9 lines long.  I get the following linter error:

```
client\src\app.scss
  9:16  error  Values for property 'font-size' may not use px units  property-units
```

The error it is reporting is clearly on line 5, not line 9.  I have other (much larger) files where the linter isn't even recording an existing line number.

**Comment by battletoilet** [comment]
Yeah i have the same exact problem, it is giving me errors on lines that don't exist.

**Comment by bgriffith** [comment]
CRLF line endings right?

**Comment by adrianhall** [comment]
Yes - windows platform.

**Comment by bgriffith** [comment]
Thanks, done some testing - it appears to be an issue with the parser we use. I've filed an issue :+1:

https://github.com/tonyganch/gonzales-pe/issues/152

**Comment by battletoilet** [comment]
Hey Guys, we would **absolutely** love to use this in our project but the strange line numbers really prevent us depending on sass-lint. I also noticed that the parser you guys use has a lot of open issues and i'm concerned that they wont fix this anytime soon. Is there an alternative solution?

**Comment by DanPurdy** [comment]
We hear you. We currently have our hands tied by our AST though, there's not really another AST suitable either as we need to support both `Sass` and `scss` which rules out post-css etc.

There is talk about a new AST being developed by @xzyfer in #427 which i think a few of us would be happy to contribute to once it's available to work on.

**Comment by DanPurdy** [comment]
Ok looked into fixing this and I believe I have a solution now. I've submitted a PR to the AST repo but we'll look to get this merged in to our temporary fork and released alongside our 1.6 update needs a bit more testing but I'm hopeful.

PR is here if you're interested 
https://github.com/tonyganch/gonzales-pe/pull/162

**Comment by DanPurdy** [comment]
@adrianhall @battletoilet So i believe I can confirm that this issue is resolved with the AST fix we did and also is now available in sass-lint 1.6.0 let us know if you find any other issues like this but CRLF should now be fixed for all our Windows using friends 😄
